scripts/python/migrate_analytics_status.py

```
@@ -0,0 +1,114 @@
+
+def update_record(record: Dict[str, Any]) -> bool:
+    """Update a single record with boolean computationNeeded."""
+    dynamodb = boto3.resource('dynamodb')
+    table = dynamodb.Table(TABLE_NAME)
+    
+    try:
+        # Convert boolean to DynamoDB Binary type
+        binary_true = Binary(b'\x01')  # Binary representation of true
+        
+        # Update the record to set computationNeeded to true
+        response = table.update_item(
```

Ruff (0.11.9)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)